### ISSUE TITLE
Favorites and channels lists are reversed

### ISSUE THREAD

**Issue by francislavoie** [mattermost/mattermost-mobile#7186]
I just opened the app to send a message, and I noticed that items in my favorites and channels are reversed. The one I usually have at the top is now at the bottom of the section, and so on. 

It still looks fine on the desktop app, so I have to assume this is a mobile version bug.

![Screenshot_20230304-133412.png](https://user-images.githubusercontent.com/2111701/222923039-80bfe164-7b55-4794-90cc-1e8f081196ef.png)

**Comment by amyblais** [comment]
@francislavoie What OS and version are you on? Do you see the same issue on our community server here https://community.mattermost.com/ (it is on the latest server version)?

**Comment by francislavoie** [comment]
Android 13, latest beta. Pixel 6 Pro.

Immediately after logging into the community server, it looks like this on mobile app & Firefox respectively (no problems observed):

![Screenshot_20230306-105752.png](https://user-images.githubusercontent.com/2111701/223164746-09f04901-8358-4d8e-96bc-1fb94168e142.png)

![Screenshot_20230306-105836.png](https://user-images.githubusercontent.com/2111701/223164774-161d2922-878e-4fdc-b239-d06c259faa43.png)

And then, I created a new category "Favourites" (note: I had to do this on my PC with a web browser because the app doesn't seem to let me create categories, and the web interface on Firefox didn't let me open the keyboard to set a name for a category either).

I added two channels to this category, Contributors first and Reception second. The web app looks correct, but the Android app has the category upside down.

![Screenshot_20230306-110249.png](https://user-images.githubusercontent.com/2111701/223165682-6b5aaf0b-c1a2-420e-be3d-9dfc0e5a7249.png)

![Screenshot_20230306-110257.png](https://user-images.githubusercontent.com/2111701/223165657-a23c2785-16da-44da-aabc-fb253430e716.png)
